src/test/scala/chiselTests/naming/PrefixSpec.scala

```diff
@@ -0,0 +1,120 @@
+// See LICENSE for license details.
+
+package chiselTests.naming
+
+import chisel3._
+import chisel3.aop.Select
+import chisel3.experimental.{ValName, prefix}
+import chiselTests.ChiselPropSpec
+
+class PrefixSpec extends ChiselPropSpec {
+  property("Scala plugin should interact with prefixing so last plugin name wins?") {
+    class Test extends MultiIOModule {
+      def builder() = {
+        val wire1 = Wire(UInt(3.W))
+        val wire2 = Wire(UInt(3.W))
+        wire2
+      }
+      val x1 = prefix("first") {
+        builder()
+      }
+      val x2 = prefix("second") {
+        builder()
+      }
+    }
+    aspectTest(() => new Test) {
+      top: Test => Select.wires(top).map(_.instanceName) should be (List("first_wire1", "x1", "second_wire1", "x2"))
+    }
+  }
+
+  property("Using String as prefix") {
+    class Test extends MultiIOModule {
+      def builder() = {
+        val wire1 = Wire(UInt(3.W))
+        val wire2 = Wire(UInt(3.W))
+        wire2
+      }
+      val x1 = prefix("x1") { builder() }
```

Comment:
Is this test just a special case of the one above, except the val name is the prefix name? Or alternatively put, an explicit version of the below? It feels kind of odd?